products/BellHybrid/apps/application-bell-onboarding/presenter/OnBoardingShortcutsWindowPresenter.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include "OnBoardingShortcutsWindowPresenter.hpp"
+#include "layouts/OnBoardingShortcutsLayoutProvider.hpp"
+
+namespace app::OnBoarding
+{
+    OnBoardingShortcutsWindowPresenter::OnBoardingShortcutsWindowPresenter(app::ApplicationCommon *app) : app(app)
+    {
+        initLayoutOptions();
+    }
+
+    std::vector<gui::Item *> OnBoardingShortcutsWindowPresenter::getLayouts() const
+    {
+        std::vector<gui::Item *> layouts;
+
+        for (auto const &option : layoutOptions) {
+            layouts.push_back(option);
+        }
+
+        return layouts;
```

Comment:
Why not return a copy of `layoutOptions` instead of manually creating this new vector?

Why not return `const std::vector<gui::Item *>&` which would prevent this copy since AFAIS `layoutOptions` are not changed after creation?

Reply:
Probably it is code from an old implementation. I forgot clean it up :)

Reply:
Done